slidetoact/src/main/java/com/ncorti/slidetoact/SlideToActView.kt

```diff
@@ -458,6 +460,11 @@ class SlideToActView @JvmOverloads constructor(
 
         // Make sure the position is recomputed.
         mPosition = 0
+
+        // Set state to complete if needed
+        if (mIsCompleted) {
+            setCompleteState()
+        }
```

Comment:
I believe this won't work if you call it inside `onSizeChanged`. This should be moved inside the `setCompleted` function